src/json-utils.js

```diff
@@ -108,15 +108,22 @@ function getAttributes (entity) {
         if (isEmpty(modifiedProperty)) {
           elemObj['components'][componentName] = '';
         } else {
-          elemObj['components'][componentName] = toPropString(modifiedProperty);
+          elemObj['components'][componentName] = toPropString(componentName, modifiedProperty);
```

Comment:
@Algorush would this change apply to saving other components (not just mapbox) and is that ok?

Reply:
Sorry, I didn't describe the reasons why I made this change.
The `mapbox` attribute's `style` property has a specific URL format that contains a colon (:) character. Example:
`style: mapbox://styles/mapbox/satellite-streets-v11`
The parser from json-utils and specifically from the JSON.stringlify function here (https://github.com/3DStreet/3dstreet/blob/938e55f00b4b1305edcde892851d9362875e2396/src/json-utils.js#L186) converts this property into this form in saved JSON file:
{style: {mapbox: {'//styles/mapbox/satellite-streets-v11'}}}.
This was the superficial reason why, when loading a JSON file with scene containing a mapbox component, the map was not loaded.
I came up with this quick solution, probably not the best one. In theory, it is better to check the contents of the stored property on the URL content using regEx, if the string contains the characters `://`

Reply:
Thanks. Would this have affected saving the streetmix URL in the streetmix-loader component for the same reason since it includes a colon character in the https:// ... ? Or we aren't saving that now for a different reason?

Reply:
I just looked at examples of saved JSON and only now I saw that streetmix URLs are also saved in separated form:
{'http': '//streetmix.net...'}
But now this does not affect the work, as far as I understand... Since streetmix-loader is saved as not-streetmix-loader.
So its better to use regex to check for such cases. I can make it in this branch